docs/schema-migration.md

```diff
@@ -16,15 +16,17 @@ Amend the `liquibaseDbUrl`, `liquibaseUsername` and `liquibasePassword` in `grad
 A _change log_ is a file that contains a series of _change sets_ (analagous to a transaction) which applies _change types_ (actions). You can refer to this page on liquibase on the types of [change types](https://docs.liquibase.com/change-types/home.html) that can be used.
 
 ## How to use Liquibase in Teammates
-1. Create an _XML_ change log file in `src/main/resources/db/changelog` naming convention is the `db.changelog-YYYY-MM-DD-entity.xml` e.g `db.changelog-2024-03-24-courses.xml`.
-2. Add changelog file to be included in the `db.changelog-root` as the last entry
+In Teammates, the release leader will add a folder in `src/main/resources/db/changelog` for changelogs that will used in the next release. 
+1. Create an _XML_ change log file in `src/main/resources/db/changelog/<latest-release>` naming convention is the `db.changelog-<entity>-<PR_Number>-<short_description>.xml` e.g `db.changelog-courses-2048-migrate-courses.xml`.
```

Comment:
Is PR number needed for naming? Since one file only contains one entity's change, the conflicts could be handled when merging PR into main imo. @cedricongjh what is your thought

Reply:
yup i agree, i think it's fine not to include, as the description will be enough to deconflict the changelog naming if there happens to be more than 1 changelog for an entity

Reply:
Sure I can amend this, thought it would be easier to identify which PR at a glance